steps/synopsys-vcs-sim/configure.yml

```diff
@@ -0,0 +1,40 @@
+#=========================================================================
+# Synopsys VCS -- Generic VCS Simulation
+#=========================================================================
+
+name: synopsys-vcs-sim
+
+#-------------------------------------------------------------------------
+# Inputs and Outputs
+#-------------------------------------------------------------------------
+
+inputs:
+  - adk
+  - design.vcs.v
+  - testbench.sv
+  - design.sdf
+  - design.args
+
+outputs:
+  - design.vpd
+
+#-------------------------------------------------------------------------
+# Commands
+#-------------------------------------------------------------------------
+
+commands:
+  - |
+    vcs -R -sverilog -top $testbench_name \
+        inputs/*.v inputs/*.sv \
+        inputs/adk/stdcells.v \
+        -timescale=1ns/1ps \
+        +vcs+dumpvars+outputs/design.vpd \
+        $(cat inputs/design.args 2>/dev/null || true)
+
+#-------------------------------------------------------------------------
+# Parameters
+#-------------------------------------------------------------------------
+
+parameters:
+  clock_period: 1.0
+  testbench_name: undefined
```

Comment:
Maybe in the future we can standardize on just using "tb" as the name for all testbenches. Makes it less confusing.